src/main/java/uk/gov/hmcts/ccd/data/user/UserService.java

```diff
@@ -58,22 +55,26 @@ public UserProfile getUserProfile() {
             }
         });
 
-        return createUserProfile(idamProperties, userDefault, jurisdictions);
+        return createUserProfile(idamProperties, userId, jurisdictions);
     }
 
-    private UserProfile createUserProfile(IDAMProperties idamProperties, UserDefault userDefault, List<Jurisdiction> jurisdictionsDefinition) {
+    private UserProfile createUserProfile(IDAMProperties idamProperties, String userId, List<Jurisdiction> jurisdictionsDefinition) {
 
         JurisdictionDisplayProperties[] resultJurisdictions = toResponse(jurisdictionsDefinition);
 
-        WorkbasketDefault workbasketDefault = new WorkbasketDefault();
-        workbasketDefault.setJurisdictionId(userDefault.getWorkBasketDefaultJurisdiction());
-        workbasketDefault.setCaseTypeId(userDefault.getWorkBasketDefaultCaseType());
-        workbasketDefault.setStateId(userDefault.getWorkBasketDefaultState());
-
         UserProfile userProfile = new UserProfile();
         userProfile.setJurisdictions(resultJurisdictions);
         userProfile.getUser().setIdamProperties(idamProperties);
-        userProfile.getDefaultSettings().setWorkbasketDefault(workbasketDefault);
+        try {
+            UserDefault userDefault = userRepository.getUserDefaultSettings(userId);
+            WorkbasketDefault workbasketDefault = new WorkbasketDefault();
+            workbasketDefault.setJurisdictionId(userDefault.getWorkBasketDefaultJurisdiction());
+            workbasketDefault.setCaseTypeId(userDefault.getWorkBasketDefaultCaseType());
+            workbasketDefault.setStateId(userDefault.getWorkBasketDefaultState());
+            userProfile.getDefaultSettings().setWorkbasketDefault(workbasketDefault);
+        } catch (ApiException ae) {
```

Comment:
https://github.com/hmcts/ccd-data-store-api/blob/24ef2666378acd0c9a96e9314e20e2019afeddd5/src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java#L134

```suggestion
        } catch (ResourceNotFoundException ae) {
```
otherwise we are handling also `BadRequestException` as a profile not existing?

Reply:
agreed... Changed